packages/features/bookings/Booker/components/DatePicker.tsx

```diff
@@ -35,6 +36,21 @@ export const DatePicker = ({
     shallow
   );
   const nonEmptyScheduleDays = useNonEmptyScheduleDays(schedule?.data?.slots);
+  const browsingDate = month ? dayjs(month) : dayjs().startOf("month");
+
+  const onMonthChange = (date: Dayjs) => {
+    setMonth(date.format("YYYY-MM"));
+    setSelectedDate(date.format("YYYY-MM-DD"));
+    setDayCount(null); // Whenever the month is changed, we nullify getting X days
+  };
+
+  useEffect(() => {
+    const currentMonth = dayjs().startOf("month").format("YYYY-MM");
+    const browsingMonth = browsingDate.format("YYYY-MM");
+    if (currentMonth === browsingMonth && nonEmptyScheduleDays.length === 0) {
+      onMonthChange(browsingDate.add(+1, "month"));
```

Comment:
```suggestion
      onMonthChange(browsingDate.add(1, "month"));
```

Not sure why + is needed.